### PR TITLE
Sage docs sidebar nav modifications 

### PR DESCRIPTION
**Pull request by teenwolfblitzer**
## Description
Slight modification of sidebar navigation output to improve link hierarchy. Also added a couple of stand-in styles for missing `:focus` and `:active` states and widened the sidebar on mobile devices.

## Related

- n/a

### CODE REVIEW

File: lib/sage-frontend/stylesheets/system/patterns/objects/_nav.scss [hunk 1]
```diff
@@ -21,17 +21,22 @@
     @extend %t-sage-body-med;
```
Comment (AndrwM) [inline lib/sage-frontend/stylesheets/system/patterns/objects/_nav.scss:22]: What do you think about adding a 1px or 2px bottom margin to these navigation links (`.sage-nav__link`)?

image

This is more of an aesthetic change, it gives the rollover color block a little room to breathe.

**Review by AndrwM — COMMENTED** [review]
I'm seeing this issue in the review app:

image

It may have something to do with a missing closing tag or a flex-grow issue with `.sage-content`. I can't tell from the browser.

**Comment by teenwolfblitzer** [comment]
> I'm seeing this issue in the review app:
> 
> image
> 
> It may have something to do with a missing closing tag or a flex-grow issue with `.sage-content`. I can't tell from the browser.

whoa! I totally missed seeing that somehow. will take a look.

File: lib/sage-frontend/stylesheets/system/patterns/objects/_nav.scss (hunk 1 above)
Comment (teenwolfblitzer) [reply lib/sage-frontend/stylesheets/system/patterns/objects/_nav.scss:22]: Nice! I like that, looks cleaner with the small spacing between links. I'll add that in.

I was hoping to circle back on the sidebar styling for a full refresh at some point. Chris had mocked up a version to use in the app, though I don't think it was final. Now that we're really close to landing Sage in the admin, hopefully one of us will be able to follow-up on that:

Comment (AndrwM) [reply lib/sage-frontend/stylesheets/system/patterns/objects/_nav.scss:22]: Sweet! I love this!

**Review by philschanely — APPROVED** [review]
🙌

**Pull request merged**